docker-compose.yml

```diff
@@ -32,8 +32,8 @@ services:
     container_name: pg
     image: postgis/postgis:12-3.0
     environment:
-      POSTGRES_USER: username
-      POSTGRES_PASSWORD: password
+      POSTGRES_USER: inaturalist
+      POSTGRES_PASSWORD: inaturalist
```

Comment:
Hmmm... 

My PR related to docker env is not yet reviewed/merged https://github.com/inaturalist/inaturalist/pull/3652

I didn't use this docker compose and I use a slightly different approach (declaring "docker" config files from these "example" ones)

The 2 new docker compose are in a docker folder (one for a full local env, one for an hybrid env, connected to staging)

I would advise to delete this one to avoid confusion, if you all agree with my new ones.